### ISSUE TITLE
question of scheduler in 2D-Tan

### ISSUE THREAD

**Issue by yxG1005** [microsoft/2D-TAN#135]
In the code, the scheduler is set as 

scheduler = optim.lr_scheduler.ReduceLROnPlateau(optimizer, factor=config.TRAIN.FACTOR, patience=config.TRAIN.PATIENCE, verbose=config.VERBOSE)
state['scheduler'].step(-val_state['loss_meter'].avg)

As official document of ReduceLROnPlateau, setting here means learning rate will change (lr = lr * factor) when metric didn't decrease in 20 epochs. (Equals to -val_state['loss_meter'].avg didn't decrease, equals to val loss didn't increase). However, when val loss increase, learning rate should change, this conflicted with the setting in code. I am confused with why here add "-" before val_state['loss_meter'].avg?

**Comment by Sy-Zhang** [comment]
Hi @yxG1005 , Thank you for pointing this out! You are absolutely right. The use of `-val_state['loss_meter'].avg` here is indeed a logic error. It conflicts with the default `mode='min'` of `ReduceLROnPlateau`.
